src/executor/executor.ts

```diff
@@ -54,6 +54,8 @@ export type ExecutorOptions = {
    * For more details see {@link JobStorage}. Defaults to a simple in-memory storage.
    */
   jobStorage?: JobStorage;
+  /** Timeout for waiting for at least one offer from the market  */
```

Comment:
Please provide a more user friendly description  in the second line, explaining why the user should care and what they can do with this setting, what will happen when the timeout is reached. Documenting it here is far more important than writing the article in the docs :)

Reply:
you're right, I'll add a better description